python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -1,21 +1,27 @@
 import json
 import os
 from datetime import datetime
-from pathlib import Path
 
 from airflow import DAG
-from astro import sql as aql
 from astro.constants import DEFAULT_CHUNK_SIZE, FileType
-from astro.files import File
 from astro.sql.table import Metadata, Table
+from google.cloud import storage
+from tests.benchmark import settings as benchmark_settings
+
+from astro import sql as aql
+from astro.files import File
 
 START_DATE = datetime(2000, 1, 1)
 
 
 def load_config():
-    config_path = Path(Path(__file__).parent.parent, "config.json").resolve()
-    with open(config_path) as fp:
-        return json.load(fp)
+    tmp_config = "/tmp/config.json"
+    storage_client = storage.Client()
```

Comment:
@kaxil @tatiana @dimberman 
1.  Is there a way we can do without this intermediate step of creating a file?
2. is there a better way to explicitly specify credentials?

Reply:
The following is not clear to me:

- When would we ever want to change the "benchmark config" file? Why?
- Why do changes in config need a new dockerfile? Can you link the code for that please?